styles.css

```diff
@@ -0,0 +1,270 @@
+/* Задаем основные параметры страници */
+.body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  background: #E5E5E5;
+  max-width: 1440px;
+}
+
+/* box-sizing позволяет задавать свойства
+width и height для блока, а не его содержимого, */
+* {
+  box-sizing: border-box;
+}
+
+/* Основные пораметры для header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 24px 60px 58px 42.72px;
+  align-items: center;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+}
+.header__logo {
+  width: 166px;
+  height: 51px;
+  padding-right: 203px;
+}
+.header__nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* Подгоняем элементы блока nav по размерам из проекта */
+.header__nav-modelS {
+  padding-right: 63.14px;
+}
+.header__nav-modelX {
+  padding-right: 24.86px;
+}
+.header__nav-model3 {
+  padding-right: 44px;
+}
+.header__nav-roadster {
+  padding-right: 46px;
+}
+.header__nav-link {
+  text-decoration: none;
+  color: #000;
+  padding-bottom: 6px;
+  background: none;
+}
+
+/* меняем поведение ссылки при наведении */
+.header__nav-link:hover {
+  color: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)